docs/CONFIG-VARS.md

```diff
@@ -340,7 +340,26 @@ Each server element, like `foo = {}`, can contain none, some, or all of the para
 | ssl_enforcement_enabled | Enforce SSL on connection to the Azure Database for PostgreSQL Flexible server instance | bool | true | |
 | postgresql_configurations | Sets a PostgreSQL Configuration value on a Azure PostgreSQL Flexible Server | list(object) | [] | More details can be found [here](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/howto-configure-server-parameters-using-cli) |
 
-Here is a sample of the `postgres_servers` variable with the `default` entry only overriding the `administrator_password` parameter and the `cps` entry overriding all of the parameters:
+Multiple SAS offerings require a second PostgreSQL instance referred to as SAS Common Data Store, or CDS PostgreSQL. For more information, see [Common Customizations](https://go.documentation.sas.com/doc/en/itopscdc/default/dplyml0phy0dkr/n08u2yg8tdkb4jn18u8zsi6yfv3d.htm#p0wkxxi9s38zbzn19ukjjaxsc0kl). A list of SAS offerings that require CDS PostgreSQL is provided in [SAS Common Data Store Requirements](https://go.documentation.sas.com/doc/en/itopscdc/default/itopssr/p06lfgkwib3zxbn1t6nyihexp12n.htm#n03wzanutmc6gon1val5fykas9aa). To create and configure an external CDS PostgreSQL instance in addition to the external platform PostgreSQL instance named `default`, specify "cds-postgres" as a second PostgreSQL instance, as shown in the example below.
+
+Here is an example of the `postgres_servers` variable with the `default` server entry overriding only the `administrator_password` and `postgresql_configurations` parameters, and the `cds-postgres` entry overriding none of the parameters:
+
+```terraform
+postgres_servers = {
+  default = {
+    administrator_password       = "D0ntL00kTh1sWay"
+    postgresql_configurations    = [
+       {
+         name  = "azure.extensions"
+         value = "PLPGSQL,LTREE"
+       }
+      ]
+  },
+  cds-postgres = {}
+}
+```
+
+Here is an example of the `postgres_servers` variable with the `default` server entry overriding only the `administrator_password` parameter, and the `cps` entry overriding all parameters:
```

Comment:
I don't believe the `cps` item is needed any more. This was replaced by the cds database.

Reply:
I will update the instance name reference from `cps` to `another_server` to match the existing example.

Reply:
@thpang, do you agree that it would be clearer to remove the last example referring to `cps` altogether since the last block doesn't add new information. We could move some of the parameter override examples from that block and add them to the cds_postgres block.